pkg/hubble/metrics/http/handler.go

```diff
@@ -82,7 +86,7 @@ func (h *httpHandler) Context() *api.ContextOptions {
 }
 
 func (h *httpHandler) ListMetricVec() []*prometheus.MetricVec {
-	return []*prometheus.MetricVec{h.requests.MetricVec, h.responses.MetricVec, h.duration.MetricVec}
```

Comment:
this change is ok since [the only caller of this function](https://github.com/cilium/cilium/blob/70c2da156ddf215f8436c2a9fa2751894476989b/pkg/hubble/metrics/api/api.go#L134) doesn't mutate the returned value.